lib/Frontend/FrontendOptions.cpp

```diff
@@ -98,7 +98,6 @@ bool FrontendInputs::verifyInputs(DiagnosticEngine &diags, bool treatAsSIL,
 
 bool FrontendInputs::areAllNonPrimariesSIB() const {
   for (const InputFile &input : getAllFiles()) {
-    assert(!input.file().empty() && "all files have (perhaps pseudo) names");
```

Comment:
Why did this change?

Reply:
There are now equivalent assertions inside of the InputFile abstraction. This is redundant.